shell.nix

```diff
@@ -5,7 +5,7 @@
 with pkgs;
 mkShellNoCC {
   sopsPGPKeyDirs = [
-    "./keys"
+    "${toString ./.}/keys"
```

Comment:
Does `sopsPGPKeyDirs` not take paths directly?

Reply:
I think it would, but I am too lazy to test :)